static/js/stats/stats.js

```diff
@@ -785,6 +799,7 @@ function populate_number_of_users(data) {
     // Initial drawing of plot
     draw_or_update_plot(all_time_trace, true);
     $("#all_time_actives_button").addClass("selected");
+    get_total_active_users(data);
```

Comment:
Would it be cleaner to pass `data.everyone`? It's not 100% clear to me whether this will get total users (or total active in last N days); might be worth a comment explaining that.

It's also possible that we'll want to display both numbers in the stats area; is that easy if we want to do it?

Reply:
> Would it be cleaner to pass `data.everyone`?

:+1: 

> It's not 100% clear to me whether this will get total users (or total active in last N days); might be worth a comment explaining that.

Yeah, this statistic is users that haven't been deactivated in the database and who aren't bots. So maybe a better name would be `get_total_not_deactivated_users`? Or `get_total_activated_users`? Or `get_total_users` with a note that we exclude deactivated users and bots?

> It's also possible that we'll want to display both numbers in the stats area; is that easy if we want to do it?

We currently track time series for  "active" users in the last day and 15 days, so those stats for the most recent day would be easy to pull out for the summary section.